Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { Breadcrumbs } from '@material-ui/core';
+import { useCallback } from 'react';
+import { isNil } from 'lodash';
+import concatClassNames from '../../../utils/class-names';
+import { IHaveOptionalClassName } from '../../common/Props';
+import generateId from '../../../utils/id-generator';
+import styles from './Breadcrumb.module.scss';
+import defaultKeyFunc from '../../common/colcollection-key-utils';
+
+interface IBreadcrumbProps<TValue> extends IHaveOptionalClassName {
+    id?: string;
+    items?: TValue[] | null,
+    itemFunc: (value: TValue) => React.ReactElement;
+    keyFunc?: (value: TValue) => string,
+}
+
+const Breadcrumb = <TValue extends unknown>({
```

Comment:
Is extends unknown a good practice?

Reply:
I am not sure. I thought that this is a convention when you want to create a generic component with a collection like in List.tsx. The documentation suggests using it like "unknown is the type-safe counterpart of any" ref: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-0.html#new-unknown-top-type like in our case. 
I don't have experience with typescript and I am not sure if is it ok to use it like that. @Minkov can you please advise here?